File: BusinessLogic/Questionnaires/Mapping/Model/Pregunta_Tests.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Pregunta_Tests : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_pregunta_test { get; set; }
       public string? Estado { get; set; }
       public string? Descripcion_pregunta { get; set; }
       public int? Id_test { get; set; }
       public int? Id_tipo_pregunta { get; set; }
       public DateTime? Fecha { get; set; }
       public DateTime? Created_at { get; set; }
       public DateTime? Updated_at { get; set; }
       public string? Seccion { get; set; }
       public string? Descripcion_general { get; set; }
       [ManyToOne(TableName = "Cat_Tipo_Preguntas", KeyColumn = "Id_tipo_pregunta", ForeignKeyColumn = "Id_tipo_pregunta")]
       public Cat_Tipo_Preguntas? Cat_Tipo_Preguntas { get; set; }
       [ManyToOne(TableName = "Tests", KeyColumn = "Id_test", ForeignKeyColumn = "Id_test")]
       public Tests? Tests { get; set; }
       [OneToMany(TableName = "Resultados_Pregunta_Tests", KeyColumn = "Id_pregunta_test", ForeignKeyColumn = "Id_pregunta_test")]
       public List<Resultados_Pregunta_Tests>? Resultados_Pregunta_Tests { get; set; }
   }
}

File: UI/wwwroot/Questionnaires/FrontModel/Pregunta_Tests.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
//@ts-ignore
import { Cat_Tipo_Preguntas } from './Cat_Tipo_Preguntas.js';
import { Resultados_Pregunta_Tests } from './Resultados_Pregunta_Tests.js';
import { Tests } from './Tests.js';
class Pregunta_Tests extends EntityClass {
    /** @param {Partial<Pregunta_Tests>} [props] */
   constructor(props) {
       super(props, 'EntityQuestionnaires');
       for (const prop in props) {
           this[prop] = props[prop];
       };
       this.Id_tipo_pregunta = undefined;
   }
   /**@type {Number}*/ Id_pregunta_test;
   /**@type {String}*/ Estado;
   /**@type {String}*/ Descripcion_pregunta;
   /**@type {Date}*/ Fecha;
   /**@type {Date}*/ Created_at;
   /**@type {Date}*/ Updated_at;
   /**@type {String}*/ Seccion;
   /**@type {String}*/ Descripcion_general;
   /**@type {Cat_Tipo_Preguntas} ManyToOne*/ Cat_Tipo_Preguntas;
   /**@type {Tests} ManyToOne*/ Tests;
   /**@type {Array<Resultados_Pregunta_Tests>} OneToMany*/ Resultados_Pregunta_Tests;
}
export { Pregunta_Tests };
